### DISCUSSION TITLE
After rerunning of script shipping component is not working

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1961] [Q&A]
![image](https://github.com/user-attachments/assets/107f59a6-38dd-47eb-b117-e79359a44b11)

**Comment by chaithanya1812** [comment]
![image](https://github.com/user-attachments/assets/19e2856b-5307-4d08-aee8-491e4efaf52f)
![image](https://github.com/user-attachments/assets/0c4b41f3-5273-41bd-89ef-f1bb02874bd1)
![image](https://github.com/user-attachments/assets/31a4f263-038e-4533-86b8-efe2e2b67421)

**Comment by Kurra432** [comment]
Issue resolved.
